04-lection2/02-skeleton/index.html

```diff
@@ -1,11 +1,24 @@
 <!DOCTYPE html>
 <!-- Страница со скелетоном -->
 <html lang="ru">
-    <head>
-        <link rel="stylesheet" href="../../assets/css/main.css">
-        <link rel="stylesheet" href="./skeleton.css">
-    </head>
-    <body>
 
-    </body>
-</html>
+<head>
+    <link rel="stylesheet" href="../../assets/css/main.css">
+    <link rel="stylesheet" href="./skeleton.css">
+</head>
+
+<body>
+    <div class="skeleton">
+        <div class="skeleton__header">
+            <div class="skeleton__circle"></div>
+            <div>
+                <div class="skeleton__text title"></div>
```

Comment:
title тут скорее как модификатор

Чтобы выделить элемент в отдельный бэм-блок, задаю себе вопрос - может ли он как-то использоваться отдельно, без компонента, в котором он изначально был

У `width: 172px;` сейчас очень мало стилей, правильнее его сделать как модификатор для `.skeleton__text`

```suggestion
                <div class="skeleton__text_title"></div>
```

или

```suggestion
                <div class="skeleton__text_type_title"></div>
```